src/backend/marsha/core/models/video.py

```diff
@@ -221,18 +40,6 @@ class Meta:
             )
         ]
 
-    def __str__(self):
-        """Get the string representation of an instance."""
-        result = f"{self.title}"
-        if self.deleted:
-            result = _("{:s} [deleted]").format(result)
-        return result
-
-    @property
-    def consumer_site(self):
-        """Return the consumer site linked to this video via the playlist."""
-        return self.playlist.consumer_site
-
```

Comment:
the is_ready_to_play method can also be remove as it is already on the BaseFile model.

Reply:
They don't have the same name because they don't have the same meaning. But maybe we can rename on of them and keep only one.